public/hp_heel.html

```diff
@@ -10,10 +10,77 @@
 </head>
 
 <body>
-  <div>
-    <a href="home.html"><img src="image/HPheel1.png" width="40px" height="40px" alt="sample"></a>
+  <a href="home.html"><img src="image/HPheel1.png" width="40px" height="40px" alt="sample"></a>
+  <img src="image/gauge.png"  class="ATKmeter" width="200px" height="90px" style="display: block; margin:auto">
+  <div class="meters-wrapper">
+    <div class="meter-panel" x-data="meterCtrl">
+      <div class="meter">
+        <div class="power">
+          <img src="image/needle2.png" class="needle" width="90px" style="transform: scale(-1, 1);"  x-bind:style="rotateStyle()">
+        </div>
+        <button x-on:click="increment" style="display: block; margin:auto">HP回復</button>
+      </div>
+    </div>
   </div>
-  <div class="background">still not working</div>
 </body>
-
+<script>
+  sessionStorage.setItem('test',128);
+  var testValue = sessionStorage.getItem('test');
+  console.log(testValue);
+  const tbl_hp=[
+    { temp:34,  get:0, prompt:"暑すぎない？" },
+    { temp:29,  get:20, prompt:"ちょっと暑いかも？" },
+    { temp:24,  get:50, prompt:"体にいい涼しさだ！" },
+    { temp:19,  get:30, prompt:" ちょっと寒いかも？" },
+    { temp:14,  get:10,  prompt:"寒くなぁい？暖を取り始めよう！" },
+    { temp:-273, get:1, prompt:"寒すぎる！暖かくしろ！" }
+  ]
+  const getTbl = async()=>{
```

Comment:
```suggestion
  const getTbl = async() => {
```

演算子の両隣に空白があるほうが個人的には読みやすくて好きです。